bazel/foreign_cc/BUILD

```diff
@@ -0,0 +1,82 @@
+licenses(["notice"])  # Apache 2
+
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
+load("@foreign_cc_platform_utils//:tools.bzl", "NINJA_COMMAND", "NINJA_DEP")
+
+envoy_package()
+
+cmake_external(
+    name = "ares",
+    cache_entries = {
+        "CARES_SHARED": "no",
+        "CARES_STATIC": "on",
+    },
+    cmake_options = ["-GNinja"],
+    lib_source = "@com_github_c_ares_c_ares//:all",
+    make_commands = [
+        NINJA_COMMAND,
+        NINJA_COMMAND + " install",
+    ],
```

Comment:
We need to copy pdb files on Windows debug builds; see: https://github.com/greenhouse-org/envoy/blob/66776969519e993ea022bb385947897208704abe/bazel/foreign_cc/BUILD#L55-L58

Reply:
Ack. Let's merge as-is and you can do a small diff PR for this. I can't verify these steps, so will let you folks take this on.

Reply:
Not a problem -- we can submit a followup PR that cleans up the Windows bits